unit_testing/test_labels.py

```diff
@@ -50,6 +50,7 @@ def fake_3dimage_sct():
 
 
 test_images = [fake_3dimage_sct(), fake_3dimage_sct2(), t2_img]
+test_labels = [fake_3dimage_sct(), fake_3dimage_sct2(), labels_img]
```

Comment:
And, this part is what fixes the failing test, right?

Reply:
Was not using labels as input (https://github.com/neuropoly/spinalcordtoolbox/pull/3191#discussion_r572102845)